### ISSUE TITLE
next/image optimized image does not look like original image

### ISSUE THREAD

**Issue by davidgaspardev** [vercel/next.js#27092]
### What version of Next.js are you using?

11.0.1

### What version of Node.js are you using?

15.7.0

### What browser are you using?

Safari

### What operating system are you using?

macOS

### How are you deploying your application?

yarn dev

### Describe the Bug

I am using the Next.js **Image** component and when it is reered to Image in the browser, the image is **damaged**, but when I use the normal **img** HTML element the image is normal, but the deploy requires `next/image`.

There is another image very similar to this that works normally.

<img width="311" alt="Screen Shot 2021-07-11 at 20 05 10" src="https://user-images.githubusercontent.com/37712275/125212372-5e8faf00-e283-11eb-9909-a50d46667ca0.png">

### Expected Behavior

Expected Behavior is the TeamViewer icon image normally render like AnyDesk.

### To Reproduce

```jsx
import Image from 'next/image'

export default function Test() {
  return (
    <div>
      { /** Team Viewer button (link) */}
      <div className="floating-button" style={{ backgroundColor: "var(--color-teamviewer)", right: 160 }} >
          <Image 
            src={"/images/png/teamviewer.png"}
            alt="TeamViewer"
            width={24}
            height={24}/>

        <h6>Team Viewer</h6>
      </div>

      { /** AnyDesk button (link) */ }
      <div className="floating-button" style={{ backgroundColor: "var(--color-anydesk)" }} >
        <Image 
          src={"/images/png/anydesk.png"}
          alt="AnyDesk"
          width={24}
          height={24} />

        <h6>AnyDesk</h6>
      </div>
    </div>
  );
}
```
### Assets
![anydesk](https://user-images.githubusercontent.com/37712275/125212419-a7476800-e283-11eb-864d-fcc0323abc0b.png)
![teamviewer](https://user-images.githubusercontent.com/37712275/125212420-a7dffe80-e283-11eb-97ba-4010e2bc14ce.png)

### Result
<img width="311" alt="Screen Shot 2021-07-11 at 20 05 10" src="https://user-images.githubusercontent.com/37712275/125212525-6ac83c00-e284-11eb-9184-fdf064dd35aa.png">

**Comment by styfle** [comment]
Looks like a bug with `squoosh`, the package we used to optimize images.

See https://github.com/GoogleChromeLabs/squoosh/issues/1102

**Comment by styfle** [comment]
This has been fixed in [v11.1.3-canary.46](https://github.com/vercel/next.js/releases/tag/v11.1.3-canary.46).

You can try it out today with `yarn add next@canary`, thanks!

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
